i18n/spec/utilSpec.js

```diff
@@ -1,16 +0,0 @@
-'use strict';
-
-var util = require('../src/util.js');
-
-describe('findLocaleId', function() {
-  it('should find localeId', function() {
-    expect(util.findLocaleId('', 'num')).toBeUndefined();
-    expect(util.findLocaleId('aa', 'datetime')).toBeUndefined();
-    expect(util.findLocaleId('aa', 'randomType')).toBeUndefined();
-    expect(util.findLocaleId('NumberFormatSymbols_en', 'datetime')).toBeUndefined();
-    expect(util.findLocaleId('DateTimeSymbols_en', 'num')).toBeUndefined();
-
-    expect(util.findLocaleId('DateTimeSymbols_en', 'datetime')).toBe('en');
-    expect(util.findLocaleId('NumberFormatSymbols_en_US', 'num')).toBe('en_US');
-  });
-});
```

Comment:
I removed the file as you wanted so these tests would no longer work.

On Mon, Aug 1, 2016 at 3:24 PM, Georgios Kalpakas <notifications@github.com>
wrote:

> In i18n/spec/utilSpec.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r72976317>:
>
> > -'use strict';
> > -
> > -var util = require('../src/util.js');
> > -
> > -describe('findLocaleId', function() {
> > -  it('should find localeId', function() {
> > -    expect(util.findLocaleId('', 'num')).toBeUndefined();
> > -    expect(util.findLocaleId('aa', 'datetime')).toBeUndefined();
> > -    expect(util.findLocaleId('aa', 'randomType')).toBeUndefined();
> > -    expect(util.findLocaleId('NumberFormatSymbols_en', 'datetime')).toBeUndefined();
> > -    expect(util.findLocaleId('DateTimeSymbols_en', 'num')).toBeUndefined();
> > -
> > -    expect(util.findLocaleId('DateTimeSymbols_en', 'datetime')).toBe('en');
> > -    expect(util.findLocaleId('NumberFormatSymbols_en_US', 'num')).toBe('en_US');
> > -  });
> > -});
>
> Maybe these could be moved to closureI18nExtractorSpec. It has some tests
> for findLocaleId, but there are some more here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular.js/pull/14952/files/a35c7b1049edc8033c0e3a6eaeeda6cd34c99e73..197c8608ca2d44fec56ef25cc44c847ad7884627#r72976317>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrUnhWg0QtpL3BLzHc_dOETyqEZ6h5eks5qbfOngaJpZM4JU90D>
> .
>



-- 
Michał Gołębiowski